data/json/recipes/recipe_electronics.json

```diff
@@ -2370,7 +2374,10 @@
   "components": [
     [ [ "processor", 1 ] ],
     [ [ "scrap", 2 ] ],
-    [ [ "wire", 5 ] ],
+    [
```

Comment:
I had assumed that in this recipe, it is playing the part of the antenna itself.